README.md

```diff
@@ -3,4 +3,7 @@
 ## Goal
 Our goal is to randomly generate an algorithm to scramble a Rubik's Cube, example shown below.
 
-Scramble:  U  D2  B'  F'  D  F'  D  U'  B2  R'  B'  R  D2  F2  U'  L2  U  L  R'  D2  F2  D  B2  D2  L
\ No newline at end of file
+Scramble:  U  D2  B'  F'  D  F'  D  U'  B2  R'  B'  R  D2  F2  U'  L2  U  L  R'  D2  F2  D  B2  D2  L
+
+## Development
+To lint this project, run `pylint $(git ls-files '*.py')`
```

Comment:
Will I run this line with the file name replacing the `*`, or do I run it exactly as written where the `'*.py'` is a regex expression?